src/tests/v1/test_production_locations.py

```
@@ -793,3 +794,107 @@ def test_claim_date_gte_greater_than_claim_date_lt(self):
+        }
+
+        self.open_search_client.index(
+            index=self.production_locations_index_name,
+            body=details_doc,
+            id=self.open_search_client.count()
+        )
+        self.open_search_client.indices.refresh(
+            index=self.production_locations_index_name
+        )
+
+        response = requests.get(
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)